docs/en/observability/synthetics-visualize.asciidoc

```diff
@@ -34,25 +34,27 @@ image::images/synthetic-app-journey.png[Synthetics app journey]
 Additional information, including the specific error and a stack trace, is provided when a step fails.
 Select *View performance breakdown* to view the waterfall chart for the synthetic monitor.
 
-[role="screenshot"]
-image::images/synthetics-waterfall.png[Synthetics waterfall chart]
+== Waterfall chart
```

Comment:
I would define anchors for the two new headings

Reply:
Thanks. Will do.